proposals/partial-properties.md

```diff
@@ -0,0 +1,142 @@
+# Partial properties
+https://github.com/dotnet/csharplang/issues/6420
+
+### Grammar
+
+The *property_declaration* grammar [(§14.7.1)](https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/classes.md#1471-general) is updated as follows:
+
+```diff
+property_declaration
+-    : attributes? property_modifier* type member_name property_body
++    : attributes? property_modifier* 'partial'? type member_name property_body
```

Comment:
1. We'll probably want to confirm whether `partial` should still be in locked position for new features (that's something we considered relaxing for existing features)
2. The structure for `method_header` is a bit different (`partial` is not inlined in the declaration, but rather inside the `method_modifiers` production). Should we spec it the same way for partial properties?
```antlr
method_declaration
    : attributes? method_modifiers return_type method_header method_body
    | attributes? ref_method_modifiers ref_kind ref_return_type method_header
      ref_method_body
    ;

method_modifiers
    : method_modifier* 'partial'?
    ;
```

Reply:
I would like to address (1) in a subsequent PR, and to do so for both properties and methods in C# 13. I recall soft approval from LDM for this, and at the time I make the change I'll also follow up by email to confirm that approval.

I would prefer to not make changes for (2) at this time, as I think the spec is conveying what it is intending to with adequate clarity, and that we'll address the concern before the spec is committed to its final state for C# 13.